xrpl/models/base_model.py

```diff
@@ -33,7 +33,7 @@
     f"(?:{_CAMEL_CASE_LEADING_LOWER}|{_CAMEL_CASE_ABBREVIATION}|{_CAMEL_CASE_TYPICAL})"
 )
 # used for converting special substrings inside CamelCase fields
-SPECIAL_CAMELCASE_STRINGS = ["NFToken"]
+SPECIAL_CAMELCASE_STRINGS = ["LPTokens", "NFToken"]
```

Comment:
super nit: this should be in the reverse order (things should be appended to the end of a list, not the beginning)

Reply:
Sure, I put it in the front so it's in alphabetical order haha